search-services/1.3/using/index.md

```diff
@@ -289,13 +289,14 @@ Single terms, phrases, and so on can be combined using "`NOT`" in upper, lower,
 These queries search for nodes that contain the terms `yellow` in any content.
 
 ```sql
-yellow NOT banana
-yellow !banana
-yellow -banana
+yellow AND NOT banana
+yellow AND !banana
+yellow AND -banana
 NOT yellow banana
 -yellow banana
 !yellow banana
 ```
+The `NOT` operator can be omitted for Share. 
```

Comment:
I don't understand how this can be the case.

Reply:
@tpage-alfresco I checked with @mjstanford. He tested the change and it looks good.

Reply:
To be clearer - I don't see how it makes sense for these queries to return the same results from Share:
```
yellow banana
NOT yellow banana
```

Reply:
I see your point now. Agree. I was focused on the change Prosune is making (which is correct), but the docs were wrong before. "yellow banana" and "NOT yellow banana" will never return the same results, which is implied by "These queries search for nodes that contain the terms `yellow` in any content."

Suggestions to be added in review comments.

Reply:
So, just to make myself clear, the meaning of the following sequence is as follows:
```yellow AND NOT banana
yellow AND !banana
yellow AND -banana
NOT yellow banana
-yellow banana
!yellow banana```

In the first 3 lines, "yellow" is being searched and "banana" is omitted. In the next 3 lines "banana" is being searched and "yellow" is omitted. 

Also, is there an instance where `yellow banana` and `NOT yellow banana` are mentioned in the same example set? I did not find one.

Reply:
@tpage-alfresco @mjstanford I have updated the suggested changes. Let me know if it looks good now or whether I have missed something.